### PR TITLE
Add datasapiens testimonial, logos, and url

### PR DESCRIPTION
**Pull request by jpohanka**
Added datasapiens testimonial, logos and URL.

### CODE REVIEW

File: _data/users.yml [hunk 1]
```diff
@@ -82,6 +82,20 @@
...
   urltext:
   url: https://datappeal.io/
+- name: datasapiens
+  anchor: datasapiens
+  logo: /assets/images/logos/datasapiens.png
+  logosmall: /assets/images/logos/datasapiens-small.png
+  testimonial: |
+    At datasapiens, we help your company to become a data-driven organization.
+    Trino is at the core of that endeavour and is our primary data retrieval
+    engine. We use it on top of various data sources from diverse industries
```
Comment (mosabua) [inline _data/users.yml:92]: industries including retail, gastronomy, travel and others.

File: _data/users.yml [hunk 2]
```diff
@@ -82,6 +82,20 @@
...
+- name: datasapiens
+  anchor: datasapiens
+  logo: /assets/images/logos/datasapiens.png
+  logosmall: /assets/images/logos/datasapiens-small.png
+  testimonial: |
+    At datasapiens, we help your company to become a data-driven organization.
+    Trino is at the core of that endeavour and is our primary data retrieval
+    engine. We use it on top of various data sources from diverse industries 
+    (i.e. retail, gastro, travel, etc.). Trino unifies a diverse set of data 
+    stores (i.e. Apache Hive, PostgreSQL, Apache Pinot, Redshift, etc.). It allows
```
Comment (mosabua) [inline _data/users.yml:94]: stores such as Apache Hive, PostgreSQL, Apache Pinot, and Redshift. It alloes

**Review by mosabua — CHANGES_REQUESTED** [review]

File: _data/users.yml [hunk 3]
```diff
@@ -82,6 +82,20 @@
     data-driven insights.
   urltext:
   url: https://datappeal.io/
+- name: datasapiens
+  anchor: datasapiens
+  logo: /assets/images/logos/datasapiens.png
+  logosmall: /assets/images/logos/datasapiens-small.png
+  testimonial: |
+    At datasapiens, we help your company to become a data-driven organization.
```
Comment (bitsondatadev) [inline _data/users.yml:90]: I almost suggested to capitalize datasapiens then looked at the website and seems like it is intentionally lower case. :)

**Review by bitsondatadev — APPROVED** [review]
Once you do Manfred's changes you're good to go!

**Review by dain — APPROVED** [review]

**Comment by mosabua** [comment]
Can we do the update in the wording to be more readable and use less brackets as suggested in my review @dain ?

**Comment by bitsondatadev** [comment]
Sorry Manfred I'll submit the fix. That was my bad.

**Pull request merged**